configure.ac

```diff
@@ -8022,21 +8027,41 @@ if test "$wxUSE_TESTS_SUBDIR" != "no"; then
     dnl Configure tests directory
     SUBDIRS="$SUBDIRS tests"
 
-    dnl Check for catch (C++ Automated Test Cases in Headers) availability.
-    AC_MSG_CHECKING([whether catch2/catch.hpp exists])
-    if ! test -f "$srcdir/3rdparty/catch/single_include/catch2/catch.hpp" ; then
-        AC_MSG_RESULT([no])
-        AC_MSG_ERROR([
-    CATCH (C++ Automated Test Cases in Headers) is required, the required file
-    $srcdir/3rdparty/catch/single_include/catch2/catch.hpp couldn't be found.
+   dnl Check for catch (C++ Automated Test Cases in Headers) availability.
+    if test "$wxUSE_CATCH2" != "builtin"; then
+        if test -n "$PKG_CONFIG"; then
+            dnl TODO: Support catch2 version 3, which does not
+            dnl ship a catch2/catch.hpp header anymore.
+            PKG_CHECK_MODULES([CATCH2], [catch2 < 3],, [
```

Comment:
We should use `CATCH2_CFLAGS` somewhere, otherwise compilation will fail if it's not installed under `/usr/include`.